zcash/wallet.go

```diff
@@ -235,7 +235,7 @@ func (w *ZCashWallet) GetTransaction(txid chainhash.Hash) (wi.Txn, error) {
 	txn, err := w.db.Txns().Get(txid)
 	if err == nil {
 		tx := wire.NewMsgTx(1)
-		rbuf := bytes.NewReader(txn.Bytes)
+		rbuf := bytes.NewReader(txn.Bytes[4:len(txn.Bytes)-15])
```

Comment:
Why are you extracting these specific bytes here. It would be good to extract this into a function which clearly labels what it's doing. I don't understand the purpose, but something like `filterTxBytes([]byte) []byte` which has documentation/tests describing its purpose.